### PR TITLE
Fix 'Using with cosmos-js' example

### PR DESCRIPTION
**Pull request by EnoRage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ const signature = signWithPrivateKey(signMessage, privateKey)
 import { signWithPrivateKey } from "@lunie/cosmos-keys"
 import Cosmos from "@lunie/cosmos-js"
 
-const privateKey = Buffer.from(...)
-const publicKey = Buffer.from(...)
+const privateKey = Buffer.from(...);
```
Comment (faboweb) [inline README.md]: ';' is not necessary. For consistency can we leave them out?

File: README.md [hunk 2]
```diff
@@ -49,15 +49,15 @@ const signature = signWithPrivateKey(signMessage, privateKey)
...
+const publicKey = Buffer.from(...);
 
 // init cosmos sender
-const cosmos = Cosmos(STARGATE_URL, ADDRESS)
+const cosmos = Cosmos(STARGATE_URL, ADDRESS);
 
 // create message
 const msg = cosmos
-  .MsgSend({toAddress: 'cosmos1abcd09876', amounts: [{ denom: 'stake', amount: 10 }})
+  .MsgSend({toAddress: 'cosmos1abcd09876', amounts: [{ denom: 'stake', amount: 10 }]});
```
Comment (faboweb) [inline README.md]: Nice find

File: README.md (hunk 1 above)
Comment (jbibla) [reply README.md]: should fail the linter, no?

Comment (faboweb) [reply README.md]: this is not a js file

Comment (EnoRage) [reply README.md]: https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required

Comment (faboweb) [reply README.md]: > https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required

This is the google style guide and not a W3C spec. We use Standard-JS as our style guide (https://standardjs.com/rules-en.html#semicolons).
I also recommend reading "Mario Kondo - Magic Cleaning" and what she says about getting rid of things "don't spark joy". :)

**Pull request closed without merging**